presto-main/src/main/java/com/facebook/presto/sql/planner/PlanPrinter.java

```diff
@@ -211,11 +220,18 @@ public static String textDistributedPlan(List<StageInfo> stages, Metadata metada
         // it's possible that some or all of them are missing or out of date.
         // For example, a LIMIT clause can cause a query to finish before stats
         // are collected from the leaf stages.
-        Map<PlanNodeId, Long> inputPositions = new HashMap<>();
-        Map<PlanNodeId, Long> inputBytes = new HashMap<>();
-        Map<PlanNodeId, Long> outputPositions = new HashMap<>();
-        Map<PlanNodeId, Long> outputBytes = new HashMap<>();
-        Map<PlanNodeId, Long> wallMillis = new HashMap<>();
+        Map<PlanNodeId, Long> planNodeInputPositions = new HashMap<>();
+        Map<PlanNodeId, Long> planNodeInputBytes = new HashMap<>();
+        Map<PlanNodeId, Long> planNodeOutputPositions = new HashMap<>();
+        Map<PlanNodeId, Long> planNodeOutputBytes = new HashMap<>();
+        Map<PlanNodeId, Long> planNodeWallMillis = new HashMap<>();
+
+        Map<PlanNodeId, Map<String, Long>> operatorTotalDrivers = new HashMap<>();
```

Comment:
In order to compute collisions or input standard deviation, I need to compute stats for each operator that was generated for given `PlanNode`. For instance:
```
- InnerJoin[("orderkey" = "orderkey_0")][$hashvalue, $hashvalue_7] => [orderkey:bigint, $hashvalue:bigint, orderkey_0:bigint, $hashvalue_7:bigint] 
        Cost: 32.52%, Output: 6001215 rows (206.04MB)                                                                                              
        LookupJoinOperator := Drivers: 4, Input avg.: 1500303.75 lines, Input std.dev.: 0.03%                                                      
        HashBuilderOperator := Drivers: 16, Input avg.: 93750.00 lines, Input std.dev.: 0.23%                                                      
        HashBuilderOperator := Collisions avg.: 117186.29 (99.60% est.), Collisions std.dev.: 1.56%
```
There are different number of drivers for `LookupJoinOperator` and `HashBuilderOperator`. Also, only `HashBuilderOperator` provides collisions data